docs/research/rhubarb_test/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "es5",                          
+    "module": "commonjs",                    
+    "lib": ["es6"],                     
+    "allowJs": true,
```

Comment:
On my Windows computer, I had to comment out `"allowJS" : true,` due to a permission error with a folder within rhubarb